Update index.html

Faris316 opened the pull request:
This is the first group (awe video)

Faris316 committed:
Update index.html

This is the first group (awe video)

DominiqueMakowski commented:
I think you override a lot of the changes that I did (i.e., you didn't update your document following my edits). 
Could you re-update with your edits [**this**](https://github.com/RealityBending/InteroceptionAwe/blob/main/experiment/index.html) document? thanks

Faris316 commented:
Oh i am sorry i didn't mean to do that, yes i will do it to the document you sent. Do you want me to add the things i changed or edit them in the document? for example for demographics questionnaire i have changed it, should i add the new one on top of the old one so you can see both or just remove the old one and add the new one?
